src/viam/rpc/dial.py

```diff
@@ -277,15 +279,28 @@ def free_str(self, ptr: ctypes.c_void_p):
 
 
 async def dial(address: str, options: Optional[DialOptions] = None) -> ViamChannel:
+    directory = os.path.dirname(__file__)
+    with open(os.path.join(directory, "../__init__.py"), "r") as file:
+        match = re.search(r'api_version\s*=\s*"([0-9]+\.[0-9]+\.[0-9]+)"', file.read())
+        api_version = match.group(1) if match else None
+    with open(os.path.join(directory, "../../../pyproject.toml"), "r") as file:
+        match = re.search(r'version\s*=\s*"([0-9]+\.[0-9]+\.[0-9]+)"', file.read())
+        sdk_version = match.group(1) if match else None
+
+    async def send_request(event: SendRequest):
+        event.metadata["viam-client"] = f'python;{sdk_version};{api_version}'
```

Comment:
Is there a better way to do this? I wish it could be a bit more elegant, but it does work!
I had to get the `sdk_version` from `pyproject.toml` since `__init__.py` does not have the version printed. Is that okay or is there a better way to get the version number?

Reply:
The `pyproject.toml` file was where I first wanted to put in the version, but Naveed realized that this file wasn't included when building.
I also thought about adding it to their own separate file, but we weren't sure what the best practice was. I personally like this approach, so let's see how it goes!